docs/default.nix

```diff
@@ -1,5 +1,6 @@
 {
   pkgs ? import <nixpkgs> { config.enableUnfree = true; },
+  nuschtosSearch,
```

Comment:
We can get this from pkgs now right? Or are we still waiting for https://github.com/NixOS/nixpkgs/pull/327670 to reach us?

Reply:
I don't think it has been packaged for nixos, I think it's meant to be consumed as a flake

Reply:
My mistake, I misinterpreted this thread: https://github.com/nix-community/nixvim/pull/1897#discussion_r1690408693